apps/datafeeder/project.json

```diff
@@ -19,7 +19,7 @@
           "apps/datafeeder/src/favicon.ico",
           "apps/datafeeder/src/assets",
           {
-            "glob": "*",
+            "glob": "*.json",
```

Comment:
What does this do? Do we need it?

Reply:
Fixed ! I've moved the script. See last comment.